backend/modules/parsers/multi_modal_parser.py

```diff
@@ -12,6 +12,12 @@
 from langchain_core.messages import HumanMessage
 from PIL import Image
 
+from backend.constants import (
+    MULTI_MODAL_PARSER_PROMPT,
+    MULTI_MODAL_PARSER_SUPPORTED_FILE_EXTENSIONS,
+    MULTI_MODAL_PARSER_SUPPORTED_IMAGE_EXTENSIONS,
+    MULTI_MODAL_PARSER_SUPPORTED_PDF_EXTENSION,
+)
```

Comment:
Not sure why this was done only for multimodal parser, specifically constants for managing extensions.

Reply:
Since this PR had changes related to this parser, I moved those constants to a constants.py file.